_data/simple-icons.json

```diff
@@ -5666,7 +5666,7 @@
         },
         {
             "title": "Honor",
-            "hex": "FFFFFF",
+            "hex": "000000",
             "source": "https://www.hihonor.com"
```

Comment:
Vector logos are available at https://www.hihonor.com/global/brand-guideline11/basics/logo. Could also be added as guidelines. According to this document, `#000000` is the correct color to use in this case!